EleventhLesson/task3/Main.java

```diff
@@ -0,0 +1,39 @@
+package task3;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class Main {
+    public static void main(String[] args) {
+        String file = "E:\\JetBrains\\AN11_Android\\NinthLesson\\task3\\Text.txt";
+        String fileBad = "E:\\JetBrains\\AN11_Android\\NinthLesson\\task3\\BadWord.txt";
+        String text = Utils.readFile(file);
+        String badText = Utils.readFile(fileBad);
+        Pattern patternOffer = Pattern.compile("[\\n]");
+        String[] offer = patternOffer.split(text, 0);
+        String[] badOffer = new String[0];
+        int k = 0;
+        for (int i = 0; i < offer.length; i++) {
+            MyThreadWord myThreadWord = new MyThreadWord(badText, offer[i]);
+            myThreadWord.run();
```

Comment:
start не run чтобы запустить в отдельном потоке
```suggestion
            myThreadWord.start();
```